attestation-verifier/src/go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/open-edge-platform/trusted-compute/attestation-verifier/src
 
-go 1.23.7
+go 1.23.9
```

Comment:
It needs update in attestation_verifier_inside_container.sh. Go package is downloaded for docker build container